msteams-platform/sbs-gs-notificationbot.yml

```diff
@@ -209,52 +208,49 @@ items:
 
     1. Run the `Invoke-Webrequest -Method POST -URI http://localhost:3978/api/notification` command.
```

Comment:
Can you change this instruction, so non-Windows users also can trigger an event? I'd suggest:

1. Go to **Terminal** > **New Terminal** to open another terminal in VS Code
2. Run this command: `curl -X POST http://localhost:3978/api/notification`

Also, can you add a short explanation why the reader has to do this step? Something like, "To trigger an event that send a notification to Teams, we are manually sending a HTTP POST request via curl command in this exercise."

Reply:
Updated as suggested